website/landingpage/dev/src/assets/js/app.js

```diff
@@ -154,10 +154,42 @@ function removeActiveState(arr) {
 
 // BLOG
 // use history api back() instead of standard link if coming from overview page
-if (jQuery('.back-to-blog').length && document.referrer.indexOf('/blog/overview') !== -1 && window.history) {
-  jQuery('.back-to-blog').on('click', function(e) {
+if ($('.back-to-blog').length && document.referrer.indexOf('/blog/overview') !== -1 && window.history) {
+  $('.back-to-blog').on('click', function(e) {
     e.preventDefault();
     e.stopPropagation();
     history.back();
   });
-}
\ No newline at end of file
+}
+
+//variables for load more blogs functionality
+var loadMoreBlogsBtn = $('#load-more-blogs-btn');
+var backToTopBtn = $('#back-to-top-btn');
+var filesAmount = $('#blog-chunks-data').data("chunk-total"); //count amount of all blogs files
+var chunksMinBlogLoadAmount = $('#blog-chunks-data').data("chunk-step"); //amount of blogs to be visible on first load
+var u = 0;
```

Comment:
What does `u` mean?
From readability point of view, could you provide more semantic naming?